packages/workflow/src/documents.ts

```diff
@@ -18,36 +18,40 @@ import {
 } from '@opencrvs/commons/types'
 import { CertifyInput, IssueInput } from './records/validations'
 
-const fetchDocuments = async <T = any>(
-  suffix: string,
-  authHeader: IAuthHeader,
-  method = 'GET',
-  body: string | undefined = undefined
-): Promise<T> => {
-  const result = await fetch(`${DOCUMENTS_URL}${suffix}`, {
-    method,
+export async function uploadFileToMinio(
+  fileData: string,
+  authHeader: IAuthHeader
+): Promise<string> {
+  const suffix = '/upload'
+  const request = {
+    method: 'POST',
     headers: {
       ...authHeader,
       'Content-Type': 'application/json'
     },
-    body
-  })
+    body: JSON.stringify({ fileData: fileData })
+  }
+  const result = await fetch(`${DOCUMENTS_URL}${suffix}`, request)
   const res = await result.json()
-  return res
+  return res.refUrl
 }
 
-export async function uploadBase64ToMinio(
+export async function uploadSVGToMinio(
```

Comment:
@euanmillar CI is complaining this export is unused
```suggestion
async function uploadSVGToMinio(
```